.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
+**
+!collector/
+!falcosecurity-libs/
+!CMakeLists.txt
```

Comment:
If we move the definition of the project to the `collector/CMakeLists.txt` file and move the falco directory into collector/ as well, we should be able to directly use that directory as the context for the image build. However, when I tried it, it give me a weird error locally and I don't want this PR to get out of hand, so we could consider doing that in a follow up.